cli/tests/unit/console_test.ts

```diff
@@ -1351,3 +1351,45 @@ unitTest(function inspectIterableLimit(): void {
     `Map { "a" => 1, "b" => 2, ... 1 more items }`,
   );
 });
+
+unitTest(function inspectProxy(): void {
+  assertEquals(
+    Deno.inspect(
+      new Proxy([1, 2, 3], { get(): void {} }),
+      { showProxy: false },
```

Comment:
Don't specify the option here, so we it effectively tests that the default is false.